### PR TITLE
[Feature] - Added Buttons component in the library

### PR DESCRIPTION
**Pull request by parthpandyappp**
### Directory structure and file justification
Added a new directory named alerts under which markup has been written in `buttons.html` and the stylesheet file is written in `buttons.css`. The stylesheet file `buttons.css` only consists of the styles related to buttons i.e it is a feature-specific stylesheet file that is later getting integrated with the main stylesheet file in the root directory i.e `styles.css`

### Explicit integration of stylesheet file
As we see, many open-source CSS libraries generally have a CDN link that is embedded and linked to the head section of any markup file. Similarly, a single main styles.css file is responsible to provide access to all the other stylesheets.

## About this PR
This PR adds a Button component of different types.

### CODE REVIEW

File: buttons/buttons.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (sonishreyas) [inline buttons/buttons.html:8]: You can add author tag here

**Review by sonishreyas — COMMENTED** [review]
Looks good to me

File: buttons/buttons.html (hunk 1 above)
Comment (parthpandyappp) [reply buttons/buttons.html:8]: Done.

**Review by yashpatel1797 — COMMENTED** [review]
Looks good to me.

**Pull request merged**